integration-tests/mtls-certificates/pom.xml

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>quarkus-integration-tests-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>quarkus-integration-test-mtls-certificates</artifactId>
+    <name>Quarkus - Integration Tests - mTLS Client Certificate tests</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy</artifactId>
```

Comment:
This is not important, but out of curiosity: why not RESTEasy Reactive? It's preferred REST stack.

Reply:
@michalvavrik Copy and paste :-), first I tested it all in `integration-tests/vertx-http`, changes some of the existing tests there and then decided to copy it all into a new test module :-)